### PR TITLE
View products front end

### PR DESCRIPTION
**Pull request by YunCYang**
image

### CODE REVIEW

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Please fix your header alignment

**Comment by YunCYang** [comment]
image
Fixed header position and size

File: server/index.js [hunk 1]
```diff
@@ -29,6 +29,17 @@ app.get('/api/products', (req, res, next) => {
     .catch(err => next(err));
 });
 
+app.get('/api/products/:productId', (req, res, next) => {
+  const sql = `
+    select *
+      from "products"
+     where "productId" = ${req.params.productId}
```
Comment (thebearingedge) [inline server/index.js]: This opens you to a SQL injection attack. Be sure to use a parameterized query.

Comment (thebearingedge) [reply server/index.js]: Also, please do not implement more than is required for the current feature.

**Review by thebearingedge — CHANGES_REQUESTED** [review]

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**